js/samples/README.md

```diff
@@ -1,114 +1,4 @@
 # Sample Library
 
-The following samples highlight key capabilities of the Teams AI SDK.
+You can find the sample library in the [getting started docs](../../getting-started/02.SAMPLES.md).
```

Comment:
Do we need to update / delete the equivalen content in the root readme about capabilities as well? Or are they different enough to keep both?

<img width="529" alt="image" src="https://github.com/microsoft/teams-ai/assets/14900841/81c62fe1-02f5-4922-9f8f-f4f861a2b560">

Reply:
The capabilities can be a sample repo specific. The getting start "sample" docs is to give a high level overview.